_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: heuristics
+title: Participatory design&#58; inclusivity heuristics
+permalink: /resources/participatory-design-inclusivity-heuristics/
+redirect_from:
+  - /participatory-design-inclusivity-heuristics/
+description: A set of heuristics for ensuring that a participatory design effort is proceeding in an inclusive way.
+---
+
+<h1>Participatory Design: Inclusivity Heuristics</h1>
+
+<p>The following indicators can be used to help determine if a <a href="{{site.baseurl}}/fundamentals/participatory-design">participatory design</a> process is proceeding in a way that encourages input from stakeholders and users. Indicators are spread across three broad categories: <a href="#account-for-context">Account for context</a>, <a href="#encourage-participation">Encourage participation</a>, and <a href="#inform-and-share-understanding">Inform and share understanding</a>.</p>
+
+<table>
+  <thead>
+    <tr>
+      <th scope="col" class="col-indicator">Indicator</th>
+      <th scope="col" class="col-why-it-matters">Why it matters</th>
+      <th scope="col" class="col-good-signs">Good signs</th>
+      <th scope="col" class="col-warning-signs">Warning signs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="4" id="account-for-context"><p>Account for context</p></th>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Values</p></th>
+      <td class="col-why-it-matters"><p>Values tie people together in ways that are difficult to break, especially when they’re not made explicit. Accounting for values helps teams approach design honestly and respectfully.</p></td>
```

Comment:
What does `values` mean here? Do you mean, `beliefs about what's important in life`? Or `what matters for the project`? Or `what behaviors are okay or not okay`?

Reply:
Fantastic question. In some ways I wanted to be intentionally ambiguous here. I suppose the most pragmatic thing for 18F engagements are: "What matters for the success of the project/product."  ...these are pretty close to design principles, in my mind.